### ISSUE TITLE
Midterm Peer Review-yz2772

### ISSUE THREAD

**Issue by WubbaLubba-DubDub** [mchao409/ORIE-4741-Class-Project#9]
**Summary**
This project is about predicting the number of bikes that are rented per station for a specific station at a specific hour. The two datasets they planned to use are NYC bike sharing data and SF Bay Area bike sharing data.

**Things I like**
1. I really like that you guys used Google Map API to generate the correct zip code based on stations' latitude and longitude. 
2. The graphs in the analysis part of your report clearly showcase the relationship between precipitation, temperature, and day of the week vs. number of bike rented. 
3. I think its a great idea to build multiple models to fit the data using different sets of features. In this way, it is better to see how features impact the prediction in a given model, how they correlate to each other, and what changes in results can be expected by adding or removing some of the features. Overall, this is a great way to do feature selection. 

**Things to be improved**
1. So far you have shown the visualizations on date, precipitation, and temperature. However, since you are prediction bike rent based on station and hour, I think it is also important to include visualizations regarding these features in your report. I feel like station/location and hour might play a more important role than weather in your dataset.
2. In your report you ran a preliminary regression analysis, but you did not report the values for MSE, MAE, or RMSE for your training and test set. I think it is necessary to take a look at these numbers to see if the model is underfitting or overfitting. Also, you could include some statistical results for your regression model such as R-squared and p-value to determine if your features accounted for the majority of the changes in your data and whether they are statistically significant or not.
3. I think besides regression model you could also try some clustering models to cluster your stations based on their locations. Another thing I want to point out is that some station locations and hours may be strongly correlated. For example, if station A is located in downtown SF, and station B is in a residential area to the south of downtown, then during morning rush hours station B will almost certainly see an increase in number of bikes rented while at the same time station A will have basically 0 bike rented because people who rent a bike at station B rides it to work, and almost nobody rents a bike at station A because nobody lives there. And during afternoon rush hours, the situation will be the opposite. Also, you guys may want to take the BART/metro stations and bus stations locations into consideration because to my knowledge many people ride a bike from home to the nearby public transportation stations everyday, so any bike stations that are next to these public transportation stations might see more increase during rush hours than other bike stations.
